utils/checks.stanza

```diff
@@ -1037,16 +1059,14 @@ public pcb-check resistor-component (r:JITXObject) :
         locators =             [r]
       )
 
-
-
   #CHECK(
-    condition =            has-property?(r.resistor),
+    condition =            has-property?(r.resistance),
```

Comment:
this will cause anything with a resistance property assigned to call this check vs an resistor. desired behavior is for only resistor components to be checked

Reply:
Same reason as above -- the actual code does not run unless the `resistor` property exists on the component in question.